user/apps.py

```diff
@@ -1,5 +1,11 @@
+"""
+UserConfig class.
```

Comment:
This should be the description of the module. I know it is difficult since it only contains one class. 
One suggestiion would be to write "Contains a class for the configuration of the user app."